### DISCUSSION TITLE
Attributter - Formål - purpose_of_use

### DISCUSSION THREAD

**Discussion by eirikbroen** [NorskHelsenett/Tillitsrammeverk#11] [General]
Dette attributtet diskuterte vi endel i pilot Oslo Kommune/HSØ, kom der frem til at det i legevaktsammenheng var lite verdifullt å avgi noen annen verdi enn "Legekonsultasjon". Ser [EHDSI](http://docs.oasis-open.org/security/xspa/v1.0/saml-xspa-1.0-cs01.html#_Toc234556166) spesifikasjon lister ganske få mulige verdier. Naturlig nivå å legge seg på? Dette er nok også en verdi som ikke finnes direkte i EPJ og som vi bør tilby eksempel på mapping av hvis det skal være obligatorisk.

**Comment by eirikbroen** [comment]
I diskusjoner knyttet til implementasjon Gerica har Oslo foreslått å benytte IPLOS tjenestetyper. HL7-kodeverket det refereres til er ikke noe som finnes i kommunale EPJ-systemer i dag. Er det slik at gitt kodeverket over er det naturlig å mappe alle de foreslåtte IPLOS tjenestetypene til _Healthcare Treatment_ (TREATMENT)?

| Verdi                                                  | Kodeverk                                                        |
|--------------------------------------------------------|-----------------------------------------------------------------|
| 07 Avlastning – utenfor institusjon                    | [9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8) |
| 08 Avlastning – i institusjon                          |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 08.1 Avlastning i institusjon – rullerende             |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 08.2 Avlastning i institusjon – ikke rullerende        |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 15 Helsetjenester i hjemmet                            |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 17 Dagopphold i institusjon                            |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)| 
| 18 Tidsbegrenset opphold – utredning/behandling        |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 19 Tidsbegrenset opphold – habilitering/rehabilitering |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)| 
| 20 Tidsbegrenset opphold – annet                       |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
|  21 Langtidsopphold i institusjon                      |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 22 Nattopphold i institusjon                           |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|

**Reply by steinarnoem** [reply]
Slik jeg tolker IPLOS verdiene forteller de noe annet enn purpose of use. 

IPLOS verdiene forteller oss noe om pasienten, mens "purpose of use" sier noe om helsepersonellet.

Jeg lurer på om dette egentlig er to forskjellige verdier - hvor IPLOS i informasjonsmodellen beskriver pasientens treff med behandlingsinstitusjonen, mens "purpose_of_use" sier noe om hva helsepersonellets skal bruke helseopplysningene til (formålet med tilgangen).

Nå er jeg på skikkelig tynn is, men jeg antar at man på et sykehjem for eksempel kan ha forskjellige kombinasjoner av IPLOS/PoU, feks:

- HDECD og 21 ("håndtering av informasjon om avdøde" og "Langtidsopphold i institusjon")
- COC og 15 ("koordinering av helsetjenester" og "Helsetjenester i hjemmet")

Etter min forståelse er det ikke vanlig i norske journalsystem å skille mellom ulike formål på denne måten.  En mulighet hadde vært at helsepersonellet selv må angi sitt "formål" i HelseID eller KJ, men det vil måtte avveies opp mot frustrasjonen helsepersonell opplever rundt et større antall antall klikk og handlinger.

**Reply by eirikbroen** [reply]
Veldig gode spørsmål @steinarnoem, la meg forsøke å beskrive noen av tankene som ligger til grunn for dette forslaget. Håper også at dette er en diskusjon flere vil delta i da jeg oppfatter at dette er et sentralt attributt. 

Som du sier er _tjenestene_  IPLOS kodene representerer knyttet til pasienten. Pasienten har et vedtak på at en virksomhet (f.eks Ammerud sykehjem) skal levere en gitt tjeneste. Dette representerer en kobling til virksomhet. Helsepersonellet som utfører diverse oppgaver relatert til leveransen av en tjeneste er knyttet til både pasienten som mottar tjenesten og virksomheten. Tanken er da at de tjenestetypene som er nevnt i første innlegg representerer helsehjelp i form av en behandling og at det er i kontekst av å yte denne tjenesten (gi behandling) at helsepersonellet har et tjenstlig behov for innsyn i helseopplysninger. Når det gjelder Gerica danner kobling til en tjeneste for en pasient grunnlaget for tilgang til pasientopplysninger. Hvis personellet er satt som ansvarlig eller utførende på en tjeneste har personellet tilgang til pasient og helseopplysninger. Det betyr at disse kodene er relativt enkelt tilgjengelige i EPJ og har en sterk knytning til tilgangsstyring. 

Det er som du skriver vanlig at pasienter kan ha flere tjenester, men gitt en filtrering på tjenester som innebærer helsehjelp og at premisset for tilgang er at det ytes en form for behandling knyttet til tjenesten er det ikke nødvendigvis et problem. Så lenge pasienten har _en_ tjeneste som innebærer helsehjelp og personellet selv vurderer det dithen at det er i relasjon til behandling forespørsel om innsyn skjer så er vi på trygg grunn. Her er det viktig å huske på individansvaret. Selv om vi bygger god systemstøtte rundt tilgang til helseopplysninger vil vi alltid måtte basere oss på at det er helsepersonellet selv som skal ta den endelige vurderingen og som har et individuellt juridisk ansvar. Om vi velger å legge tjenestetype til grunn for å implisitt utlede formål fremfor å kreve at personellet eksplisitt velger formål er et interessant spørsmål.

**Reply by OK-SverreM** [reply]
Jeg kan være enig i den distinksjonen, Steinar. Jeg har også lurt på om '_purpose_of_use_' er riktig sted å putte IPLOS-koden. 
Men da har vi behov for et annet attributt som kan uttrykke "helsetjenesten som er grunnlag for oppslaget, og som knytter den aktuelle pasienten til det angitte behandlingsstedet og helsepersonellet". 

Som du også påpeker, er det ikke tilrettelagt for at det som ønskes uttrykt gjennom '_purpose_of_use_' kan avgis fra EPJ-systemene vi jobber med i behovskøelement #27. 
Det bør være veldig høy terskel for å be helsepersonell gjøre et manuelt valg av klassifisering. Det skaper en ekstra terskel og mulig usikkerhet rundt bruk av tjenesten, og all erfaring tilsier at manuell klassifisering blir 'feil' i en ikke ubetydelig andel av tilfellene.
**_Konklusjon_**? '_purpose_of_use_' bør/kan ikke være obligatorisk i første versjon av Tillitsrammeverket.

**Reply by jndid** [reply]
En ting jeg lurer på her, som også er et mer generelt spørsmål: Hvordan er vurderingene rundt dataminimering? Informasjon om helsepersonellets rolle, formål osv. er jo helseopplysninger, så da bør man begrense hvor mye informasjon som utveksles, og informasjonen skal ha et klart formål. Ut fra det eirikbroen skriver over er det tjenestetypen som gir grunnlag for tilgang til opplysninger lokalt. Er det rimelig at man må oppgi mer informasjon for å få tilgang til opplysninger fra andre? Og et relatert spørsmål: Hvorfor skal modellen inneholde attributter som ikke er obligatoriske (eller er de egentlig "obligatoriske" der de er relevante)?
(Mulig dette er dumme spørsmål, jeg har ikke deltatt i diskusjonen før :-).

**Reply by steinarnoem** [reply]
Hei! Her var det flere spørsmål, jeg skal forsøke å svare så godt jeg kan. Så kan kanskje andre korrigere meg eller gi utfyllende informasjon hvis jeg bommer eller glemmer noe..
 
Informasjonsmodellen i dagens utkast er en speiling av attributtene EHDI (MyHealth), men med noen tilpassinger. Konseptuelt sett er den ikke fryktelig ulik det som tidligere er diskutert i sektoren. Vi jobber nå med å tilpasse modellen slik at vi når målet om deling av helseopplysninger via KJ dokumentdeling i september.

Dataminimering er en naturlig del av informasjonsmodelleringen, slik at vi sørger for at alle informasjonselementer som beskriver grunnlaget for tilgangen har et klart og tydelig formål.

Noen informasjonselementer er valgfrie på grunn av at de ikke er relevante for alle typer helsevirksomheter. Så kan man sikkert diskutere hvorvidt de bør være med dersom de er valgfrie eller ikke. Sektoren har foreløpig ikke ansett dette som problematisk, og anser informasjonen som nødvendig for å utføre tilgangsstyring og logging av tilgangene. Helsepersonellets relasjon til pasienten ansees ikke som mer sensitiv enn helseopplysninger som skal deles,  derfor er innstillingen at det ikke er problematisk å overføre denne informasjonen. Samtidig må vi unngå å overføre informasjon som kan være ansett som spesielt sensitivt, f.eks. overgrepssaker, og stille krav om at informasjonen ikke skal benyttes til å overvåke ansatte i andre virksomheter.

Identitetsattributtene som overføres i tokens beskriver helsepersonellets rolle/funksjon/kontekst i sin relasjon til pasienten. Denne autorisasjonsinformasjonen kan brukes til tilgangsstyring/tilgangskontroll ved forespørsler om helseopplysninger, og for logging av tilganger som er gitt.

**Comment by steinarnoem** [comment]
I utkastet har vi definert et attributt som heter "facility_type", og som angir helsetjenestetypen som leveres av virksomheten. Dette attributtet er definert i EHDSI, men vi har foreslått å utvide gyldige kodeverk med koder fra volven. 

Kunne vi ha innført et nytt attributt for å uttrykke verdien med koder fra 9151 som heter "health_care_type", "admission_type" eller noe lignende?

F.eks:
````JSON
"admission_type": {
     "code": "15",
     "text": "Helsetjenester i hjemmet",
     "system": "urn:oid:x.x.x.x.x.9151",
     "assigner": "volven.no"
}
````

**Reply by OK-SverreM** [reply]
Ja, jeg støtter forslaget om et nytt attributt. Ønsker dog et annet navn enn _admission_type_, som smaker litt for mye av sykehus / institusjon.
I VKP FHIR-representasjon er disse tjenestene representert gjennom ressursen _episodeOfCare_. https://www.hl7.org/fhir/episodeofcare.html
Det kan bli litt voldsomt å kalle dette _episode_of_care_type_, men kanskje det forenklede _care_type_ kan gjøre susen?
